app/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "arrowParens": "always"
+}
```

Comment:
Are these the settings we use on the portal API/UI repos? If not, let's bring that over.

Reply:
is the one from portal backend